cmd/ocm-backplane/console/console.go

```diff
@@ -68,6 +67,8 @@ var (
 	// For mocking
 	createClientSet = func(c *rest.Config) (kubernetes.Interface, error) { return kubernetes.NewForConfig(c) }
 	createCommand   = exec.Command
+
+	configDirectory string
```

Comment:
I'm presuming this is in the pull secret config directory. Can it have a relevant variable name to avoid confusion with the actual BP config dir?

Reply:
Rename as `pullSecreConfigDirectory`